net/nginx-util/Makefile

```diff
@@ -0,0 +1,70 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nginx-util
+PKG_VERSION:=1.0
+PKG_RELEASE:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/nginx-util
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=Web Servers/Proxies
+  TITLE:=Builder of LAN listen directives for Nginx
+  DEPENDS:=+libstdcpp +libubus +libubox +libpthread
```

Comment:
Out of curiosity, does this build with uClibc++?

Reply:
Right now it is not working, it complains about missing array, mutex, string_view and thread libraries. At least in my test, maybe I am doing it wrong: Do I have to include only uclibc++.mk and use $(CXX_DEPENDS) in the Makefile?